ports/raspberrypi/boards/adafruit_macropad_rp2040/mpconfigboard.h

```diff
@@ -0,0 +1,7 @@
+#define MICROPY_HW_BOARD_NAME "Adafruit Macropad RP2040"
+#define MICROPY_HW_MCU_NAME "rp2040"
+
+#define MICROPY_HW_NEOPIXEL (&pin_GPIO16)
+
+#define DEFAULT_I2C_BUS_SCL (&pin_GPIO25)
+#define DEFAULT_I2C_BUS_SDA (&pin_GPIO24)
```

Comment:
I don't see these pins in my copy. Have you pushed the latest files?

Reply:
ok i just commited again.